crates/util/Cargo.toml

```diff
@@ -32,6 +32,9 @@ git2 = { workspace = true, optional = true }
 dirs = "3.0"
 take-until = "0.2.0"
 
+[target.'cfg(windows)'.dependencies]
+tendril = "0.4.3"
```

Comment:
Hmm, it looks like WTF-8 is a superset of UTF-8, so maybe we could avoid a dependency on `tendril` and instead, in `try_from_bytes` we could convert the bytes to `&str` first and then go to PathBuf/Path?

Reply:
In most cases it will still work. However, we can create an incorrect filename as UTF-16 in NTFS (just as we can create an incorrect filename as UTF-8 in ext4). WTF-8 is needed to encode that.  
`[u8]` is a superset of `str`, should we always use `std::str::from_utf8`?

Reply:
Ah, you're right. My bad.